lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,55 @@
+require 'httparty'
+
+class EdamamApiWrapper
+  BASE_URL = "https://api.edamam.com/search?"
+  KEY = ENV["APP_KEY"]
+  APP_ID = ENV["APP_ID"]
+
+  def self.list_recipes(word)
+    url = BASE_URL + "q=#{word}" + "&app_id=#{ APP_ID }" +
+    "&app_key=#{ KEY }" + "&to=100"
+    data = HTTParty.get(url)
+    recipe_list = []
+    # binding.pry
+    if data["hits"]
+      data["hits"].each do |hit_data|
+        recipe_list << create_recipe(hit_data)
+      end
+      return recipe_list
+    else
+      return []
+    end
+  end
+
+  def self.show_recipe(id)
+    uri_first = URI.encode("http://www.edamam.com/ontologies/edamam.owl#recipe")
```

Comment:
I like how you broke up the URI so you only have to pass the id portion via params.